### PR TITLE
Missed slash in "if" and minor formatting

### PR DESCRIPTION
**Pull request by Roog**
Hi!
Some suggested changes. Hope you like it!

These links seems broken?!:

### Why:

Missed a "/"-slash in URL configuration reference

### What's being changed:

- Installing packages from ither organizations had a missplaced slash "/" for the IF
- Missed two quotes on see "Installing packages...organizations"
- Intendention on "publishConfig" was not aligned

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by muslim199 — COMMENTED** [review]
Hahah

**Review by corlisa66 — CHANGES_REQUESTED** [review]
received_662954071321561
received_679004142742408

**Comment by janiceilene** [comment]
Thank you so much for this PR @Roog! I'll get this triaged for review ⚡

File: content/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ To ensure the repository's URL is correct, replace REPOSITORY with the name of t
 
 You can install packages from {% data variables.product.prodname_registry %} by adding the packages as dependencies in the *package.json* file for your project. For more information on using a *package.json* in your project, see "[Working with package.json](https://docs.npmjs.com/getting-started/using-a-package.json)" in the npm documentation.
 
-By default, you can add packages from one organization. For more information, see [Installing packages from other organizations](#installing-packages-from-other-organizations)
+By default, you can add packages from one organization. For more information, see "[Installing packages from other organizations](#installing-packages-from-other-organizations)".
```
Comment (rachmari) [inline content/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages.md:139]: This is super counterintuitive, and I'll admit that I actually prefer the British usage, but the GitHub docs use the American usage of the period and it needs to go inside the quotes here. Here's a reference [Blog](https://www.grammarly.com/blog/period/) from Grammarly that explains it a bit. 

```suggestion
By default, you can add packages from one organization. For more information, see "[Installing packages from other organizations](#installing-packages-from-other-organizations)."
```

**Review by rachmari — COMMENTED** [review]
@Roog thank you so much for finding and fixing this bug! 🙌  Also thank you for aligning the other changes with our style guide. 💚:octocat: This is great. I had one super tiny comment, but as soon as that is fixed, I'll get this merged! 🚀

**Review by rachmari — APPROVED** [review]

File: content/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages.md (hunk 1 above)
Comment (Roog) [reply content/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages.md]: Perfect! I agree. Just noticed the use of quotes in the rest of the document and my OCD kicked in 😆 
Thanks!

**Pull request merged**